index/functions.c

```diff
@@ -357,9 +357,15 @@ static int op_delete_thread(struct IndexSharedData *shared,
 static int op_display_address(struct IndexSharedData *shared,
                               struct IndexPrivateData *priv, int op)
 {
-  if (!shared->email)
-    return FR_NO_ACTION;
-  mutt_display_address(shared->email->env);
+  // enum QuadOption ans = query_quadoption("How many roads must a man walk down, before they call him a man?", NeoMutt->sub, "quit");
```

Comment:
What is going on here? 😆

Reply:
hehe, oops!
Some of my debugging escaped.
I needed to display lots of long strings that I could wrap.